src/frformat/geo/coordonnees_gps_francaises.py

```diff
@@ -149,3 +128,29 @@ def is_point_in_france(coordonnees_xy: tuple[float, float]) -> bool:
         },
     },
 ]
+
+
+def create_polygons() -> list[BaseGeometry]:
+    geoms = [region["geometry"] for region in FRANCE_BOUNDING_BOXES]
+    polys = [shape(geom) for geom in geoms]
+    return polys
+
+
+polygons: list[BaseGeometry] = create_polygons()
```

Comment:
```suggestion
POLYGONS: list[BaseGeometry] = create_polygons()
```

C'est plutôt d'usage de nommer les constantes en majuscule (sous-entendu : ne pas modifier)